### DISCUSSION TITLE
Excel vs sheets

### DISCUSSION THREAD

**Discussion by landaj16** [PWNAGERobotics/ScoutingPASS#79] [Q&A]
Hello,

My team is looking to use this and am looking for some clarification. 

If we want to use the scanner to not worry about wifi does that mean the data will only go into excel? Or is there now a way to get the scanner to send data to Google sheets?  We would really like the data to be sent to Google sheets but from reading more carefully it looks like using the scanner does not allow for that. Please let me know your thought! Thank you!

**Comment by fuddster** [comment]
See my reply to your message on the other thread, but TLDR: We don't support scanning QR codes directly to Google Sheets, but technically it should be possible if you have some programmers that want to tackle that problem.
